_aggregations/metric/extended-stats.md

```diff
@@ -9,8 +9,41 @@ redirect_from:
 
 # Extended stats aggregations
 
-The `extended_stats` aggregation is an extended version of the [`stats`]({{site.url}}{{site.baseurl}}/query-dsl/aggregations/metric/stats/) aggregation. Apart from including basic stats, `extended_stats` also returns stats such as `sum_of_squares`, `variance`, and `std_deviation`.
-The following example returns extended stats for `taxful_total_price`:
+The `extended_stats` aggregation is a more comprehensive version of the [`stats`]({{site.url}}{{site.baseurl}}/query-dsl/aggregations/metric/stats/) aggregation. As well as the basic statistical measures provided by stats, `extended_stats` calculates:
+
+- Sum of squares
+- Variance
+- Population variance
+- Sampling variance
+- Standard deviation
+- Population standard deviation
+- Sampling standard deviation
+- Standard deviation bounds:
+  - Upper
+  - Lower
+  - Population upper
+  - Population lower
+  - Sampling upper
+  - Sampling lower
+
+The standard deviation and variance are population statistics; they are always equal to the population standard deviation and variance, respectively.
+
+The `std_deviation_bounds` object provides values representing an interval of plus or minus a number of standard deviations from the mean (two by default). This object is always displayed, but is meaningful only for normally distributed data. It's up to you to verify that your data set has a normal distribution.
+
+## Parameters
+
+The `extended_stats` aggregation takes the following parameters: 
+
+| Parameter | Required/Optional | Data type             | Description |
+| :--       | :--               | :--                   | :--         |
+| `field`   | Required          | String                | Name of the field for which the extended stats are returned. |
+| `sigma`   | Optional          | Double (non-negative) | One-tailed interval represented by the `std_deviation_bounds` object. Default is 2. |
```

Comment:
In the definition of `std_deviaion_bounds` above, we say +- 2sigma (meaning, a two-tailed interval).  Should this be
```suggestion
| `sigma`   | Optional          | Double (non-negative) | The number of standard deviations above and below the mean used to calculate the `std_deviation_bounds` interval. Default is `2`. |
```
Also, should we say what happens if you set `sigma` to 0?

Reply:
Yes, I think that's a better way to say it, especially since the readers aren't necessarily statisticians.

0 is a legitimate value for `sigma`, just not an interesting one. It reduces the bounds interval to zero -- the bounds are all equal to the mean. IMO this is the expected result and doesn't need to be documented, but I'll defer if anyone thinks otherwise.